src/qt/transactiondesc.cpp

```diff
@@ -10,11 +10,13 @@
 #include "transactionrecord.h"
 
 #include "base58.h"
+#include "consensus/consensus.h"
 #include "main.h"
 #include "script/script.h"
 #include "timedata.h"
 #include "ui_interface.h"
 #include "util.h"
+#include "wallet/db.h"
```

Comment:
The question is not whether you can compile with it or not, the question is whether qt/transactiondesc.cpp is using anything declared in wallet/db.h or not. It's better to indicate dependencies explicitly.

Try commenting ```#include "main.h"``` and ```#include "wallet/db.h"``` and build again.
 
You will notice that 
```
#include "coins.h"
#include <boost/foreach.hpp>
```
is missing but...yeah, you're right. wallet/db.h doesn't seem to be needed anymore.
None of nWalletDBUpdated, CDBEnv, bitdb and CDB is used here.
I am very sorry for putting it there unnecessarily. It seems that you removed it and I added it back in a rebase by accident.
Mhmm, I'll review #6068 in case something similar has happened there.